### PR TITLE
[python3] save restore window position

### PR DESCRIPTION
**Pull request by pedromorgan**
save

### CODE REVIEW

File: src/pyspread.py [hunk 1]
```diff
@@ -186,17 +205,19 @@ def safe_mode(self, value):
...
     def on_nothing(self):
         """Dummy action that does nothing"""
-
         pass
 
+    def on_fullscreen(self):
+        if self.windowState() == Qt.WindowFullScreen:
+            self.setWindowState(Qt.WindowMaximized)
+        else:
+            self.setWindowState(Qt.WindowFullScreen)
```
Comment (manns) [inline src/pyspread.py]: self._old_window_state = self.WindowState()

File: src/pyspread.py [hunk 2]
```diff
@@ -186,17 +205,19 @@ def safe_mode(self, value):
...
 
     def on_nothing(self):
         """Dummy action that does nothing"""
-
         pass
 
+    def on_fullscreen(self):
+        if self.windowState() == Qt.WindowFullScreen:
+            self.setWindowState(Qt.WindowMaximized)
+        else:
```
Comment (manns) [inline src/pyspread.py]: self._old_window_state = self.windowState()

File: src/pyspread.py [hunk 3]
```diff
@@ -186,17 +205,19 @@ def safe_mode(self, value):
...
-        self.workflows.file_quit()
-        event.ignore()
 
     def on_nothing(self):
         """Dummy action that does nothing"""
-
         pass
 
+    def on_fullscreen(self):
+        if self.windowState() == Qt.WindowFullScreen:
```
Comment (manns) [inline src/pyspread.py]: self.setWindowState(self._old_window_state)

**Comment by pedromorgan** [comment]
@manns fixed..

**Comment by manns** [comment]
> @manns fixed..
Could you please add the changes to the merge commit?

**Comment by pedromorgan** [comment]
apologies, not sure what am doing here.. is it ok now ?

**Comment by manns** [comment]
Yes that did it. Thank you so much.

**Pull request merged**